app/components/public/ticket-list.js

```diff
@@ -40,6 +40,25 @@ export default Component.extend(FormMixin, {
     ) > 0;
   }),
 
+  donationTickets: computed(function() {
```

Comment:
you are not watching the property which you are using here, initially this.data will be unresolved, your computed property probably never gets updated hence nothing works.